src/main/java/io/github/thebusybiscuit/cscorelib2/protection/modules/FunnyGuildsProtectionModule.java

```diff
@@ -0,0 +1,48 @@
+package io.github.thebusybiscuit.cscorelib2.protection.modules;
+
+import io.github.thebusybiscuit.cscorelib2.protection.ProtectableAction;
+import io.github.thebusybiscuit.cscorelib2.protection.ProtectionModule;
+import me.angeschossen.lands.api.integration.LandsIntegration;
+import me.angeschossen.lands.api.land.Area;
+import me.angeschossen.lands.api.land.LandWorld;
+import me.angeschossen.lands.api.role.enums.RoleSetting;
+import net.dzikoysk.funnyguilds.FunnyGuilds;
+import net.dzikoysk.funnyguilds.system.protection.ProtectionSystem;
+import org.bukkit.Location;
+import org.bukkit.OfflinePlayer;
+import org.bukkit.entity.Player;
+import org.bukkit.plugin.Plugin;
+
+public class FunnyGuildsProtectionModule implements ProtectionModule {
+
+	private final Plugin plugin;
+
+	public FunnyGuildsProtectionModule(Plugin plugin) {
```

Comment:
Can you please annotate the parameter with lombok's NonNull?

Reply:
No problem. I was writing the code basing on how other modules are written - they also lacks lomok's NonNull.